src/test/regress/multi_schedule

```diff
@@ -48,83 +49,142 @@ test: multi_partitioning_utils multi_partitioning replicated_partitioned_table
 # connection" errors on CI. Requires investigation before doing them in
 # parallel again.
 test: subquery_basics
+test: check_cluster_state
```

Comment:
why do we have some many check_cluster_state? isn't it enough to have a single one at the end?